bitwarden_license/src/Scim/Startup.cs

```diff
@@ -75,6 +77,8 @@ public void ConfigureServices(IServiceCollection services)
             config.Filters.Add(new LoggingExceptionHandlerFilterAttribute());
         });
         services.Configure<RouteOptions>(options => options.LowercaseUrls = true);
+
+        services.AddScoped<IPostUserCommand, PostUserCommand>();
```

Comment:
As we continue to add commands we probably want to put these in a separate function to make it a bit clearer what's going on.

Reply:
I've created `ScimServiceCollectionExtensions` to add all the command dependencies there.